### DISCUSSION TITLE
Control Room | Inverter Documentation

### DISCUSSION THREAD

**Discussion by giscus** [shellhub-io/shellhub#3761] [General]
# Control Room | Inverter Documentation

http://localhost:3000/docs/apps/control-room

**Comment by BartuCulha** [comment]
Hhaha

**Comment by gustavosbarreto** [comment]
Fixed by #3764

For everyone reading this, we want to emphasize that we are an open community welcoming broad discussion and collaboration. Contributions are highly encouraged, and behaviors like this will not be tolerated in our community.

The user responsible has been banned permanently from our community.
